http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -446,12 +454,17 @@ trait ClusterComponent extends LeoComponent {
         .update(
           initBucket.map(_.toUri),
           serviceAccountKey.map(_.id.value),
-          cluster.dataprocInfo.googleId,
-          cluster.dataprocInfo.operationName.map(_.value),
-          cluster.dataprocInfo.stagingBucket.map(_.value),
+          cluster.dataprocInfo.map(_.googleId),
+          cluster.dataprocInfo.map(_.operationName.value),
+          cluster.dataprocInfo.map(_.stagingBucket.value),
           Timestamp.from(Instant.now)
         )
 
+    def clearAsyncClusterCreationFields(cluster: Cluster): DBIO[Int] =
+      findByIdQuery(cluster.id)
+        .map(c => (c.initBucket, c.serviceAccountKeyId, c.googleId, c.operationName, c.stagingBucket, c.dateAccessed))
+        .update(None, None, None, None, None, Timestamp.from(Instant.now))
```

Comment:
`Instant.now` is side effecting..So ideally it should be passed in as parameter

Reply:
Yeah.. or use a DB function `current_timestamp`

Reply:
Slick apparently doesn't support updates with non-"pure" values. So you can't do:
```
update TABLE set ts = current_timestamp
```
I instead changed these methods to take an `Instant` as a parameter, and had callers do stuff like:
```
for {
  now <- IO(Instant.now)
  _ <- dbRef.inTransactionIO { _.clusterQuery. clearAsyncClusterCreationFields(cluster, now) }
} yield ()
```